index.d.ts

```diff
@@ -2513,6 +2516,10 @@ declare namespace Shopify {
   interface ITRansactionReceipt {
     testcase: boolean;
     authorization: string;
+    balance_transaction?: {
```

Comment:
Is there any documentation available for this?

Reply:
the docs say it could have anything in there depending on the gateway so maybe it would be better to make it more flexible

Reply:
updated it to be:

```typescript
interface ITRansactionReceipt {
    testcase: boolean;
    authorization: string;
    balance_transaction?: { [k: string]: any };
}
```